p2p/peer_behaviour_test.go

```diff
@@ -0,0 +1,23 @@
+package p2p
+
+import (
+	"net"
+	"testing"
+)
+
+func TestStorePeerBehaviour(t *testing.T) {
```

Comment:
Would be good to test that there could be more than one error per peer and that the lists are empty with a fresh instance.